react/features/base/conference/actions.js

```diff
@@ -736,9 +753,21 @@ export function toggleAudioOnly() {
  * @param {string} subject - The new subject.
  * @returns {void}
  */
-export function setSubject(subject: String) {
-    return {
-        type: SET_CONFERENCE_SUBJECT,
-        subject
+export function setSubject(subject: string = '') {
+    return (dispatch: Dispatch<*>, getState: Function) => {
+        const { conference } = getState()['features/base/conference'];
+
+        if (conference) {
+            dispatch({
+                type: SET_PENDING_SUBJECT_CHANGE,
+                subject: undefined
```

Comment:
Is there any noticeable jank for momentarily clearing the subject from redux?

Reply:
No. We can even leave the value in the store.